app/src/main/res/layout/welcome.xml

```diff
@@ -12,8 +12,8 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingStart="28dp"
-    android:paddingEnd="28dp">
+    android:paddingStart="@dimen/home_outer_margin"
+    android:paddingEnd="@dimen/home_outer_margin">
```

Comment:
Do not use already defined values for this as it created dependencies on one another.

Create new dimens values like `home_welcome_outer_padding`

Reply:
Ok sir I'll see it and use different dimens.